src/System.Diagnostics.Debug/src/Configurations.props

```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       netstandard-Unix;
       netstandard-Windows_NT;
+      netcoreapp1.2corert-Unix;
+      netcoreapp1.2corert-Windows_NT;
```

Comment:
That I agree, but this is how it was added in master so I'm just replicating it. I will have to do a pass on this anyway on master, so for now I'll check in this as is, and go and fix all of the netcoreapp1.2corert configurations as part of the UWP work.